trace/LogCorrelation.md

```diff
@@ -0,0 +1,63 @@
+# Log Correlation
+
+Log correlation is a feature that inserts information about the current span into log entries
+created by existing logging frameworks.  The feature can be used to add more context to log entries,
+filter log entries by trace ID, or show log entries as annotations on a trace.
```

Comment:
"log entries as annotations on a trace"

This is a possibility but do we want to encourage this usage? I think we should keep it out.

Reply:
I think the assumption that this would be used to decorate traces might be faulty. it would be more likely to have correlated queries especially as tracing uis are not as good at filtering logs (which can be extremely volumnous) than logging uis.

Reply:
so I agree we should not anchor people with the idea that this will result in log entries as span annotations. eg leave it out or rename to language that is correlation in nature vs the word annotation which is historically the name of a part of a span